test_activator.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
```

Comment:
i'm adding live test config in #65 . probably can leverage it to test the mcp directly